package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@zwave-js/server",
-  "version": "1.10.3",
+  "version": "1.11.4",
```

```suggestion
  "version": "1.10.4",
```
We just want to bump the patch version, right?

---

doh, thank you!